features/layout-instability.yml

```
@@ -1,5 +1,5 @@
 name: Layout instability
-description: The `layout-shift` performance entry and `LayoutShift` API measures the layout stability of web pages based on movements of the elements on the page.
+description: The `layout-shift` performance entry and `LayoutShift` API measures the layout stability of web pages based on movements of the elements on the page. Cumulative Layout Shift (CLS) is a common metric for perceived stability calculated from the Layout Instability API.
```

For consistency with the other suggestions, mostly.

```suggestion
description: The `layout-shift` performance entry and `LayoutShift` API measures the layout stability of web pages based on movements of the elements on the page. They're used to calculate Cumulative Layout Shift (CLS), a common metric for perceived stability.
```